<ui version="4.0" >
 <class>ProcWizardChoicesPage</class>
 <widget class="QWidget" name="ProcWizardChoicesPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="4" column="1" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Requirements</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="whatsThis" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This is the maximum number of choices that the voter is allowed to choose.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Ma&amp;ximum:</string>
        </property>
        <property name="buddy" >
         <cstring>maximumSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="whatsThis" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">This is the minimum number of choices that the voting is required to select.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>&amp;Minimum:</string>
        </property>
        <property name="buddy" >
         <cstring>minimumSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="maximumSpinBox" />
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="minimumSpinBox" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>&amp;New choice:</string>
         </property>
         <property name="buddy" >
          <cstring>newchoiceLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="newchoiceLineEdit" >
         <property name="whatsThis" >
          <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Enter a new choice here.  Once you have entered a new choice, click 'Add' to add it to the list.&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addPushButton" >
         <property name="text" >
          <string>&amp;Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removePushButton" >
         <property name="text" >
          <string>&amp;Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="choicesListWidget" >
       <property name="whatsThis" >
        <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Displays the list of current choices.  To remove a choice, select the choice and click on the 'Remove' button.&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:10pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;">&lt;span style=" font-weight:600;">Create procedure ballot.&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&amp;Ballot question:</string>
     </property>
     <property name="buddy" >
      <cstring>questionTextEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QTextEdit" name="questionTextEdit" >
     <property name="whatsThis" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Enter the ballot question.  This is the text that will be displayed to the voter along with the choices.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="html" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>questionTextEdit</tabstop>
  <tabstop>newchoiceLineEdit</tabstop>
  <tabstop>addPushButton</tabstop>
  <tabstop>removePushButton</tabstop>
  <tabstop>choicesListWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>newchoiceLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>271</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel" >
     <x>362</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
